src/avatar/Avatar.js

```diff
@@ -0,0 +1,131 @@
+import React, { PropTypes } from 'react';
+import {
+  View,
+  Image,
+  StyleSheet,
+  TouchableOpacity,
+} from 'react-native';
+
+import Text from '../text/Text'
+import normalize from '../helpers/normalizeText'
+
+const Avatar = ({
+  component,
+  width,
+  height,
+  onPress,
+  onLongPress,
+  containerStyle,
+  source,
+  avatarStyle,
+  roundAvatar,
+  title,
+  titleStyle,
+  overlayContainerStyle,
+  activeOpacity,
+}) => {
+  if (!width && !height) {
+    width = 34
+    height = 34
+  } else if (!width) {
+    width = height
+  } else if (!height) {
+    height = width
+  }
+
+  const styles = StyleSheet.create({
+    container: {
+      paddingTop: 10,
+      paddingRight: 10,
+      paddingBottom: 10,
+      borderBottomColor: '#ededed',
```

Comment:
That's unnecessary. I'll there caused I copy pasted the style from another component. 
I'll remove it.